### DISCUSSION TITLE
rate limit: get the current rate

### DISCUSSION THREAD

**Discussion by nitzan-blink** [taskforcesh/bullmq#2594] [Q&A] [answered]
Hi,
assuming I create a queue with a `limiter` option, is it possible to get the current rate of a queue?
e.g. `max` is set to 10, and the current rate is 4, I want to know it's 4.
I saw the option of [getRateLimitTtl](https://docs.bullmq.io/guide/rate-limiting#get-queue-rate-limit-ttl), but per the documentation, it only tells you if the queue has rate limit defined for it.

thanks.

**Comment by manast** [comment]
We are only counting the number of jobs in the current duration window, so we do not store information about the rate at which jobs are been processed.

**Reply by nitzan-blink** [reply]
> We are only counting the number of jobs in the current duration window

and how can I get that number?

**Reply by manast** [reply]
There is no api to get it but you can find it in Redis key: ```bull:queueName:limiter```

**Reply by manast** [reply, marked as answer]
You could divide this counter by the PTTL  to get an average though...
